frontend/src/MuiForms/CustomTextInput.tsx

```diff
@@ -36,6 +37,7 @@ export default function CustomTextInput(props: CustomTextInputProps) {
       <Typography fontWeight='bold'>{label}</Typography>
       <TextField
         size='small'
+        id={id}
```

Comment:
See https://github.com/gchq/Bailo/pull/944/files#r1423934882

Reply:
This is to make it so we have unique `id` props to use when we call `cy.get()`. We _might_ be able to dynamically create data-set names but it's up to you really.

Reply:
Not sure it's worth the time to fiddle with, happy to leave it as is for now